Seneca-CDOT/telescope
# Add password in conn string for backup script

## Issue This PR Addresses

Fixes #3459.

## Type of Change


## Description

This PR adds the password to connect to Postgres through a connection string.

## Steps to test the PR

1. Start the services.
2. Enter some information into a few tables.
3. Login into the `pg-backup-cron-job` container (`docker exec -it pg-backup-cron-job sh`).
4. Instead of waiting for the cron job to be executed, you may execute the file with the following command: `/etc/periodic/daily/create-backup`. After the command has been executed, there should be a file in `/var/opt/pg_dumps`, similar to `2022-04-08-02231.dump`, make a note of its name.
5. After getting the backup file, modify the database by deleting those tables with data.
6. To restore those tables, run the `restore-backup` file with the following command: `./restore-backup /var/opt/pg_dumps/2022-04-08-02231.dump`, for example.


## Checklist
